compiler/testData/codegen/box/jvm8/treeMapBridge.kt

```diff
@@ -1,4 +1,5 @@
 // TARGET_BACKEND: JVM
+// IGNORE_BACKEND_FIR: JVM_IR
```

Comment:
I'm not sure why this test was passing. The generated IR is not correct, e.g., the fake override for `IntervalTreeMap.get` takes an argument of type `Any?` instead of `String`.